CMakeLists.txt

```diff
@@ -361,7 +367,7 @@ message(STATUS "----------------------------------------------------------------
 message(STATUS " Libraw ${RAW_LIB_VERSION_STRING} configuration            <http://www.libraw.org>")
 message(STATUS "")
 
-if(OpenMP_FOUND)
+if(OPENMP_SUPPORT_CAN_BE_COMPILED)
```

Comment:
OK fine, I had a look at the configure script of the libraw repo and it doesn't enforce things either. So let's keep your original proposal but let's use `find_package(OpenMP)` so that OpenMP is optional even if requested, in line with the configure script.

Reply:
Updated now